pkgs/desktops/xfce/core/xfce-polkit/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, gtk3
+, glib
+, libxfce4ui
+, polkit
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xfce-polkit";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://github.com/ncopa/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-Vk5WaoPC3V/CGZSB3rVoQnl/776YXU5VIZusvrVz8jo=";
+  };
+
+  nativeBuildInputs = [
+    glib
+    pkg-config
+  ];
+
+  buildInputs = [
+    libxfce4ui
+    gtk3
+    polkit
+  ];
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "A simple PolicyKit authentication agent for XFCE";
+    homepage = "https://github.com/ncopa/xfce-polkit";
+    license = [ licenses.gpl2Plus ];
+    platforms = platforms.unix;
+    maintainers = teams.xfce.members;
```

Comment:
We will appreciate if you can also also list yourself in maintainers

Reply:
done